app/lib/bk/compat/parsers/circleci/orbs/docker/install_docker_compose.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module CircleCISteps
+      # Handles the translation of docker installation commands
+      class DockerOrb
+        def translate_docker_install_docker_compose(config)
+          install_dir = config.fetch('install-dir', '/usr/local/bin')
+          version = config.fetch('version', 'latest')
+          [
+            '# Instead of installing docker-compose in a step, ' \
+            'we recommend your agent environment to have it pre-installed',
```

Comment:
why have this particular message split into two lines whereas all the others are in a single line?

Either have all of them be in a single line or split them in the resulting script as well:

```suggestion
            '# Instead of installing docker-compose in a step, ',
            '# we recommend your agent environment to have it pre-installed',
```